Sources/BowEffects/Data/Internal/Token.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+internal class Token {
+    private let id: Double = Date().timeIntervalSince1970
+}
+
+extension Token: Hashable {
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
+    }
+}
+
+extension Token: Equatable {}
+
+func ==(lhs: Token, rhs: Token) -> Bool {
+    return lhs === rhs
```

Comment:
maybe, in this case, it is better to check just the `id` - for example, if we cached it or similar? - I want to say if exists a case where changing the object (persist/retrieve, copy...) and do not change the value (id)

Reply:
It is a reference value and the intention is to check they are actually the same object, not with the same id.